rts/at/bridge/pom.xml

```diff
@@ -258,6 +258,20 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <jvm.args.modular>--add-modules java.se
```

Comment:
Similar to the txframework comment, I noticed this does not have `--add-modules=java.se` like several of the other situations and rather has `--add-modules java.se` - is it interchangeable to configure in such a way?

Reply:
it works without '=' also